irohad/ordering/impl/ordering_service_impl.cpp

```diff
@@ -34,8 +35,11 @@ namespace iroha {
     }
 
     void OrderingServiceImpl::onTransaction(
-        const model::Transaction &transaction) {
-      queue_.push(transaction);
+        const shared_model::detail::PolymorphicWrapper<
+            shared_model::interface::Transaction> transaction) {
+      auto proto = static_cast<const shared_model::proto::Transaction *>(
```

Comment:
Is it necessary to cast it to proto here? A collection of pointers to interface should be possible to use.

Reply:
There's no other way to cast it from PW